appservice/src/createAppService/createAppService.ts

```diff
@@ -24,15 +25,26 @@ export async function createAppService(
     credentials: ServiceClientCredentials,
     subscriptionId: string,
     subscriptionDisplayName: string,
-    showCreatingNode?: (label: string) => void): Promise<Site> {
+    showCreatingNode?: (label: string) => void,
+    advancedCreation: boolean = false): Promise<Site> {
```

Comment:
I don't see you ever passing 'advancedCreation' as true?

Reply:
advancedCreation will be passed as true in the vscode-azureappservice extension.